### ISSUE TITLE
New Test Use Case

### ISSUE THREAD

**Issue by alex4200** [cnr-ibf-pa/hbp-bsp-issues#345]
## New Use Case
  
Aspect | Detail
------------ | -------------
Summary | This is an example of a complete descriptiob to describe a new use case with all the required information. The information below is purely fictional.
Usecase Group | Test _(Group like 'Single Cell Experiments', 'Trace Analysis' etc.)_ 
Expert | @alex4200 _Main developer for this use case_
Scientific User | @alex4200  _The science user that can assess the scientific correctness of this use case_
Deadline | 15.11.2018
Target audience | SP6 technical team _('Scientists', 'General users', 'BBP only', whatever...)_
Target interface | GUI _('GUI', 'Notebook', 'Nootbook with some GUI', 'Service', ...)_
HPC Requirements | NSG _(or what other system might be used)_
Dependencies | BluePy, RTNeuron _(e.g. BBP packages the use case relies on)_
Nominal runtime | 15 minutes
  
**Remove anything that does not apply**
## List of features

- Simulation of inductivity between two neurons.
- User able to select two or more neurons (up to 10)
- User able to save scenario
- User able to export scenario to Jupyter notebook
- User able to use morphology from Knowledgegraph or upload own morphologies.
- User able to select physics model or upload own model. 
- Simulation don on NSG
- Simulation playback shown as graphical representation
- Nice to have: Graphical coloring can be adjusted
- Nice to have: Audio transformation of simulation

### Tasks

- [ ] Write NSG API functions
- [ ] Make user select neurons
- [ ] Export to Jupyter
- [ ] Implement physics models (waiting on models from someone)

  
## Acceptance Tests

- User logs into BSP
- User selects this use case
- User able to select neurons from Hippocampus
- Alternative: User able to upload her/his own neurons
- User able to select the mode and the simulation parameters via GUI
- User able to start the simulation, done on NSG in background
- User able to view the simulation as playback, able to pause, rewind and continue the results
- User able to save scenario and export simulation. 
- Alternative: User able to download scenario

  
### System

- Installation of Voxcell view required, min. version 5.2.1
- Installation of ABC required. 
- Access to NSG required
  
### Performance

- NSG simulation must not take more than 5 minutes to complete
- Total user case should be done in 15 minutes
